### DISCUSSION TITLE
Your Google Developer is a Hacker and has ruined by Google Chromebook

### DISCUSSION THREAD

**Discussion by Myidentitywasstolen** [community/community#118600] [Code Security]
### Select Topic Area

Bug

### Body

Dec. 30-2023 I received a cyber attack and changed my operating system.  This person is terrorising me still to this day.  I have called the police, FBI, RCMP, Bell, Virgin Plus and nothing gets fixed.  I did find this information today.  Please put my laptop back to the way it was.  I am not rich and cannot buy another.  My identity was also stolen.  This person should be in jail and I want to move on with my life.  I don't like people who are thieves and scammers you belong in jail.  PLEASE HELP ME!

**Comment by ebndev** [comment]
Hi [Myidentitywasstolen](https://github.com/Myidentitywasstolen),

We're sorry to hear about your situation. The GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. 

We are unable to provide support here but we'd recommend contacting a local IT specialist for assistance if you think your devices are affected by viruses or malware.

I've removed your private information from [this post](https://github.com/orgs/community/discussions/118600) as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).

Thank you for your understanding.
